### DISCUSSION TITLE
new vs old labels where is labels?

### DISCUSSION THREAD

**Discussion by sacedronfly** [systemed/tilemaker#724] [General]
hello everyone
i have both version of tilemaker.
Could it be that the new version of TileMaker doesn't create a vector file with labels?
When i using old version and extract mbtiles to vector map from pbf i can render every labels of city,town village i use config-openmaptiles and process-openmaptiles.
when i using new version of tilemaker and use newest files process and config include with project opentilemap i can renderer and didnt see labels of cities,villages or towns ? its a bug?
i use the same command with both version "tilemaker --input file.pbf  --output file.mbtiles -process file_openmaptiles.lua --config config-openmaptiles.json"
i trying renderer with style for example part of file:
{
      "id": "country_label",
      "type": "symbol",
	  
      "source": "openmaptiles",
      "source-layer": "place",
      "maxzoom": 12,
      "filter": [
        "all",
        ["==", "$type", "Point"],
        ["==", "class", "country"],
        ["has", "iso_a2"]
      ],
      "layout": {
        "text-field": "{name:latin}",
        "text-font": ["Noto Sans Bold"],
        "text-max-width": 10,
        "text-size": {"stops": [[3, 12], [8, 22]]},
        "visibility": "visible"
      },
      "paint": {
        "text-color": "hsl(0, 0%, 13%)",
        "text-halo-blur": 0,
        "text-halo-color": "rgba(255,255,255,0.75)",
        "text-halo-width": 2
      }
}

whats wrong with this?

**Comment by systemed** [comment]
The first step is to examine your vector tiles to see whether they have the placenames in them. Either:

1. Install [vt2geojson](https://github.com/mapbox/vt2geojson) and use it to fetch and decompile the tiles (e.g. `vt2geojson http://localhost:8080/5/14/37.pbf`)
2. Use QGIS. Create new project; drag the .mbtiles file into it; then zoom in to the desired level and use the inspect tool to look at the attributes
3. Use the [Maplibre Inspect](https://github.com/maplibre/maplibre-gl-inspect) plugin

**Reply by sacedronfly** [reply]
where can i find information about name of road?
is it ferry class ?
 i want label of road in poland i my country.
thanks

**Reply by systemed** [reply]
Have you done any of the things I suggested? What was the output?
